src/utils/api.jsx

```diff
@@ -2,14 +2,25 @@ export const fetchImage = (username) =>
     fetch(`https://nc-be-project-news.onrender.com/api/users`)
```

Comment:
You're making two requests to the same endpoint, you already hae this information when you utilise the fetchUsers endpoint. This will rack up a big bill as your app expands (hypothetically)